citi-onboarding/celeiro-dos-bambas
# Feature/footer

## What I did

- I created the footer component
- Atualized the Strapy fields and rules

## How to test

Open the terminal and go to the folder of this project
In terminal:

1.In the folder server:
      1. `yarn install`
      2. `yarn develop`
      3. Check if you are loged
   
2. In the client folder:
    1. `yarn install`
    2. `yarn start`
